files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -328,14 +325,6 @@
     - platform == "gcp"
   notify: "gcp config changed"
 
-#
-# Make sure that the account daemon is always disabled. The override file
-# above should prevent this and this is designed to catch any corner cases.
-#
-- command: systemctl disable google-accounts-daemon.service
```

Comment:
Do we need to undo any configuration that might have previously been applied? E.g. re-enable the service (assuming it's default state is to be enabled)?

Reply:
We don't need to do anything as part of upgrade. The packaging remove the old services but does leave them masked:
```
root@gwilson-gcp:/var/tmp# systemctl list-unit-files \*google\*
UNIT FILE                        STATE
google-accounts-daemon.service   masked
google-clock-skew-daemon.service masked
google-guest-agent.service       enabled
google-instance-setup.service    masked
google-network-daemon.service    masked
google-oslogin-cache.service     static
google-shutdown-scripts.service  enabled
google-startup-scripts.service   enabled
google-oslogin-cache.timer       enabled
```

If someone ever unmasks the service then it just gets removed.